manipulation/src/main/java/org/jboss/gm/manipulation/actions/MavenPublicationRepositoryAction.java

```diff
@@ -46,9 +46,11 @@ public void execute(Project project) {
         if (uploadArchives == null) {
             project.getLogger().info("Creating uploadArchives task");
             uploadArchives = project.getTasks().create("uploadArchives", Upload.class);
+        } else {
+            uploadArchives.getRepositories().clear();
         }
 
-        Configuration config = ConfigCache.getOrCreate(Configuration.class);
+        Configuration config = ConfigCache.get(Configuration.class);
```

Comment:
Why is this changed to get?

Reply:
My mistake, I'll revert. I was trying various things and this got left behind

Reply:
fixed